libs/designsystem/src/lib/components/tabs/tabs.component.ts

```diff
@@ -9,6 +9,14 @@ import { TabsService } from './tabs.service';
   styleUrls: ['./tabs.component.scss'],
 })
 export class TabsComponent implements AfterViewInit {
+  @HostBinding('class.hide-tab-bar')
+  hideTabBar = false;
+  show() {
+    this.hideTabBar = false;
+  }
+  hide() {
+    this.hideTabBar = true;
+  }
```

Comment:
Do we need methods for this? Consider just having hideTabBar property...

Reply:
No, you never need methods if you can access properties, but it makes it simpler to show and hide

Reply:
Why is it simpler?